apps/web-giddh/src/app/new-inventory/component/bulk-stock-edit/bulk-stock-edit.component.html

```diff
@@ -1104,17 +1112,35 @@ <h3>{{ localeData?.fixed_asset_tax_inclusive }}</h3>
                                                         </text-field> -->
                                                     </div>
                                                 </td>
+                                                <!-- Custom Fileds (Menu)-->
+                                                <ng-container
+                                                    *ngIf="
+                                                        tableHeaderDynamicColumns.length &&
+                                                        row?.get('customFields')?.value?.length
+                                                    "
+                                                >
+                                                    <td
+                                                        *ngFor="let column of row.get('customFields').value"
+                                                        class="min-width-140"
+                                                    >
+                                                        <div class="border-line">
```

Comment:
```suggestion
                                                        <div class="border-line custom-padding">
```
and remove below div